src/componenets/header.js

```diff
@@ -0,0 +1,27 @@
+// TEST
+
+class foodAppHeader extends HTMLElement {
+  constructor() {
+    super();
```

Comment:
same here, you will probably have some hard times with merging, remember to w8 for kuba to merge his changes first or clear all of his files